tutorials/application_design/c++11/common.hpp

```diff
@@ -0,0 +1,97 @@
+//
+// (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+//
+//  RTI grants Licensee a license to use, modify, compile, and create derivative
+//  works of the Software.  Licensee has the right to distribute object form
+//  only for use with RTI products.  The Software is provided "as is", with no
+//  warranty of any type, including any warranty for fitness for any purpose.
+//  RTI is under no obligation to maintain or support the Software.  RTI shall
+//  not be liable for any incidental or consequential damages arising out of the
+//  use or inability to use the software.
+
+#ifndef COMMON_HPP
+#define COMMON_HPP
+
+#include <vector>
+#include <sstream>
+#include <random>
+
+#include "VehicleModeling.hpp"
+
+namespace utils {
+
+double random_range(double min, double max)
```

Comment:
I chose to avoid `<random>` because it involved a lot of set-up, but I think your proposal makes sense, specially now that I have `SrandInit`. Still, I'd not be too happy with including a `set_random_seed()` in `main` because this would be a C++-only change in the documentation but that's minimal.

While I'm always up to make things more explicit, I think we can do with the existing implicit nature for this toy example.

Reply:
I've applied these changes in commit 6be1bf1caf39582bdf72a4b9118b96bcb380ee2f.